README.md

```diff
@@ -27,41 +29,395 @@ the UKNW auth checker stub.
+    },
+    ...
+  ]
+}
+    </pre>
+    </td>
+</tr>
+
+<tr>
+<td>perfTest_500Eori</td>
+<td>Valid request with 500 EORIs matching <strong>^(GB&#124;XI)[0-9]{12}&#124;(GB&#124;XI)[0-9]{15}$</strong></td>
+<td>
+    <pre>
+{
+  "validityDate": "{{date}}",
+  "authType": "UKNW",
+  "eoris": [
+    "GB067923337129",
+    "XI041177131154142",
+    "XI675926161105887",
+    "GB889342696852",
+    "XI555810835227",
+    ...
+  ]
+}
+    </pre>
+</td>
+<td>
+    <pre>
+{
+  "processingDate": "{{dateTime}}",
+  "authType": "UKNW",
+  "results": [
+    {
+      "eori": "GB067923337129",
+      "valid": true,
+      "code": 0
+    },
+    {
+      "eori": "XI041177131154142",
+      "valid": true,
+      "code": 0
+    },
+    {
+      "eori": "XI675926161105887",
+      "valid": true,
+      "code": 0
+    },
+    {
+      "eori": "GB889342696852",
+      "valid": true,
+      "code": 0
+    },
+    {
+      "eori": "XI555810835227",
+      "valid": true,
+      "code": 0
+    },
+    ...
+  ]
+}
+    </pre>
+    </td>
+</tr>
+
+<tr>
+<td>perfTest_1000Eori</td>
+<td>Valid request with 1000 EORIs matching <strong>^(GB&#124;XI)[0-9]{12}&#124;(GB&#124;XI)[0-9]{15}$</strong></td>
+<td>
+    <pre>
+{
+  "validityDate": "{{date}}",
+  "authType": "UKNW",
+  "eoris": [
+    "GB702601139907",
+    "GB305827433009",
+    "XI569274482382127",
+    "GB823797040419",
+    "XI777849853217",
+    ...
+  ]
+}
+    </pre>
+</td>
+<td>
+    <pre>
+{
+  "processingDate": "{{dateTime}}",
+  "authType": "UKNW",
+  "results": [
+    {
+      "eori": "GB702601139907",
+      "valid": true,
+      "code": 0
+    },
+    {
+      "eori": "GB305827433009",
+      "valid": true,
+      "code": 0
+    },
+    {
+      "eori": "XI569274482382127",
+      "valid": true,
+      "code": 0
+    },
+    {
+      "eori": "GB823797040419",
+      "valid": true,
+      "code": 0
+    },
+    {
+      "eori": "XI777849853217",
+      "valid": true,
+      "code": 0
+    },
+    ...
+  ]
+}
+    </pre>
+    </td>
+</tr>
+
+<tr>
+<td>perfTest_3000Eori</td>
+<td>Valid request with 3000 EORIs matching <strong>^(GB&#124;XI)[0-9]{12}&#124;(GB&#124;XI)[0-9]{15}$</strong></td>
+<td>
+    <pre>
+{
+  "validityDate": "{{date}}",
+  "authType": "UKNW",
+  "eoris": [
+    "GB837826880909874",
+    "XI968840631629",
+    "XI436105828614",
+    "XI670738444417",
+    "XI299090776708",
+    ...
+  ]
+}
+    </pre>
+</td>
+<td>
+    <pre>
+{
+  "processingDate": "{{dateTime}}",
+  "authType": "UKNW",
+  "results": [
+    {
+      "eori": "GB837826880909874",
+      "valid": true,
+      "code": 0
+    },
+    {
+      "eori": "XI968840631629",
+      "valid": true,
+      "code": 0
+    },
+    {
+      "eori": "XI436105828614",
+      "valid": true,
+      "code": 0
+    },
+    {
+      "eori": "XI670738444417",
+      "valid": true,
+      "code": 0
+    },
+    {
+      "eori": "XI299090776708",
+      "valid": true,
+      "code": 0
+    },
+    ...
+  ]
+}
+    </pre>
+    </td>
+</tr>
+</table>
```

Comment:
Not a major comment, but would it be worth it to add the expected 500 message, if the request does not match one in the stub service?

It will be this after NP-278 is merged

```json
{
  "errorDetail": {
    "errorCode": 500,
    "errorMessage": "An internal error has occurred"
  }
}
```